### PR TITLE
Added body bag boxes, fixed some folded body bag behavior

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
Adds body bag boxes and gives folded body bags an item size.
**Screenshots**

image

### CODE REVIEW

File: Resources/Prototypes/Catalog/Fills/Boxes/medical.yml [hunk 1]
```diff
@@ -65,3 +65,25 @@
...
+    size: 30
+  - type: Storage
+    capacity: 30
+  - type: Sprite
+    layers:
+      - state: box
+      - state: bodybags
+    whitelist:
+      tags:
+        - BodyBag
```
Comment (ShadowCommander) [inline Resources/Prototypes/Catalog/Fills/Boxes/medical.yml:89]: I don't think the box should have a whitelist. None of the other boxes do unless there's something special about the box I'm missing.

**Review by ShadowCommander — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Boxes/medical.yml (hunk 1 above)
Comment (Elijahrane) [reply Resources/Prototypes/Catalog/Fills/Boxes/medical.yml:89]: Other boxes have them (light bulbs, ...) are Mirrorcult says it's fine

File: Resources/Prototypes/Entities/Objects/Specific/Medical/morgue.yml [hunk 2]
```diff
@@ -64,6 +64,11 @@
   components:
   - type: Foldable
     folded: true
+  - type: Tag
+    tags:
+    - BodyBag
+  - type: Item
+    size: 6
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Specific/Medical/morgue.yml:71]: These should be on the base copy. The only difference is this is pre-folded for reasons.

Comment (Elijahrane) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/morgue.yml:71]: If I do this this lets you pick up and drop unfolded body bags, which seems to break. Folded ones work fine

**Comment by Elijahrane** [comment]
Alright that issue is fixed by https://github.com/space-wizards/space-station-14/pull/6057

File: Resources/Prototypes/Entities/Objects/Specific/Medical/morgue.yml (hunk 2 above)
Comment (ElectroJr) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/morgue.yml]: You moved the item over, but still need to move the tag over.

**Review by ElectroJr — APPROVED** [review]

**Pull request merged**